desktop/release-notes.md

```diff
@@ -24,17 +24,42 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
-## 4.21.1
+## 4.22.0
 
-{% include release-date.html date="2023-07-03" %}
+{% include release-date.html date="2023-07-27" %}
 
-{% include desktop-install.md all=true version="4.21.1" build_path="/" %}
+{% include desktop-install.md all=true version="4.22.0" build_path="/" %}
 
 ### Upgrades
-- [Compose v2.19.1](https://github.com/docker/compose/releases/tag/v2.19.1)
+
+- [Buildx v0.11.2](https://github.com/docker/buildx/releases/tag/v0.11.2)
+- [Compose v2.20.0](https://github.com/docker/compose/releases/tag/v2.20.0)
+- [Docker Engine v24.0.5](https://docs.docker.com/engine/release-notes/24.0/#2405)
 
 ### Bug fixes and enhancements
 
+#### For all platforms
+- Extended Settings-Management features to allow IT/admins to disable Docker-Desktop "extensions".
+- Fixed a bug where enabling Kubernetes from the UI failed when the system is paused.
+- Fixed a bug where enabling Wasm from the UI failed when the system is paused.
+- Bindmounts are shown in the **Container** view.
+- You can now download Wasm runtimes when the containerd image store is enabled.
+
+#### For Mac
+
+- Fixed a bug that prevented Docker Desktop from starting. [docker/for-mac#6890](https://github.com/docker/for-mac/issues/6890)
```

Comment:
```suggestion
- Docker Desktop will use much less CPU and memory when not running containers.
```

Reply:
@doringeman I've added that it's an experimental feature. Not convinced we should add the rest of the note. It works well in a blog post, but not so well in release notes.

Reply:
Can we use `It massively reduces`?
I've heard from Chris that the same expression will be used for the blog. 😁 

Moreover, it's enabled for Windows Hyper-V (not WSL2) as well.

Reply:
Noted! Will change now